plugins/experimental/uri_signing/unit_tests/CMakeLists.txt

```diff
@@ -26,8 +26,6 @@ add_executable(
   ${PROJECT_SOURCE_DIR}/match.cc
   ${PROJECT_SOURCE_DIR}/normalize.cc
   ${PROJECT_SOURCE_DIR}/parse.cc
-  ${PROJECT_SOURCE_DIR}/timing.cc
```

Comment:
What is the point of timing.cc?

Reply:
This file doesn' t set URI_SIGNING_UNIT_TEST that I can see.  Did you build/run the unit test after you make this change?

Reply:
it builds just fine (using the autest preset).
In a previous PR I only removed the timing calls from the uri_signing.cc.  They were only returning ns level values which aren't particularly useful and junked up the uri_signing debug.  I wouldn't mind removing the rest of the references but perhaps in a later PR.